images/extension-builders/tinygo/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Tetrate
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Builder image for Envoy Wasm extensions written in Go.
+#
+
+FROM tinygo/tinygo:0.17.0
```

Comment:
[fyi] It looks like base image sets wrong permissions on files.

I can't delete cache files from Mac:

```shell
rm -rf build

rm: build/.gomodcache/github.com/tetratelabs/proxy-wasm-go-sdk@v0.0.16-0.20210310041209-1980230c2fb8/go.mod: Permission denied
rm: build/.gomodcache/github.com/tetratelabs/proxy-wasm-go-sdk@v0.0.16-0.20210310041209-1980230c2fb8/LICENSE: Permission denied
```

`getenvoy extension clean` works as expected.

Just be aware of.

Reply:
fixed in https://github.com/tetratelabs/getenvoy/pull/104/commits/b485f0571a8d6060ad883d193cd221a491ea59bc